src/clj_kondo/impl/analysis/java.clj

```diff
@@ -1,7 +1,7 @@
 (ns clj-kondo.impl.analysis.java
   {:no-doc true}
   (:require
-   [clj-kondo.impl.utils :refer [->uri update-vals]]
+   [clj-kondo.impl.utils :as utils]
```

Comment:
I opted to use an alias here (as opposed to `refer-clojure`+`exclude`) to match the usage here: https://github.com/clj-kondo/clj-kondo/blob/b03521a9a0d5217e06a13989acb17ddfea6f2d18/src/clj_kondo/impl/core.clj#L628